api/db/migrations/trip/20191125100001_create_export_view.up.sql

```diff
@@ -0,0 +1,45 @@
+CREATE MATERIALIZED VIEW trip.export AS (
+  SELECT
+    cpp.operator_id as operator_id,
+    tis.territory_id as start_territory_id,
+    tie.territory_id as end_territory_id,
+    cpp.acquisition_id::varchar as journey_id,
+    cpp.trip_id as trip_id,
+    cpp.datetime as journey_start_datetime,
+    trunc(ST_X(cpp.start_position::geometry)::numeric, cis.density::int) as journey_start_lat,
```

Comment:
```suggestion
    trunc(ST_X(cpp.start_position::geometry)::numeric, cis.density::int - 1) as journey_start_lat,
```
cf. #160 
Dans le cas où `density = NULL`, on devrait utiliser `2`. On peut patcher toutes les `density` vides avec une simple requête je pense. C'est plus simple.